services/compute/placementgroup/placementgroup.go

```diff
@@ -144,12 +144,12 @@ func getWssdPlacementGroup(s *wssdcloudcompute.PlacementGroup) (*compute.Placeme
 		Location: &s.LocationName,
 		Version:  &s.Status.Version.Number,
 		PlacementGroupProperties: &compute.PlacementGroupProperties{
-			VirtualMachines: getWssdVirtualMachineReferences(s.VirtualMachines),
-			Statuses:        status.GetStatuses(s.Status),
-			Zones:           &pgZone,
-			Scope:           pgScope,
-			StrictPlacement: pgStrictPlacement,
-			Type:            pgType,
+			VirtualMachines:     getWssdVirtualMachineReferences(s.VirtualMachines),
```

Comment:
Nit: just curious, why are we adding these extra spaces here and above. if not required, please revert them

Reply:
Fields are not aligned properly. So either VScode / go fmt has fixed the alignment. I can revert them but same auto adjustment will be done for other developers. Please let me know if alignment changes need to be reverted.